google/cloud/firestore_admin_v1/services/firestore_admin/async_client.py

```diff
@@ -273,9 +273,9 @@ async def list_indexes(
                 maximum=60.0,
                 multiplier=1.3,
                 predicate=retries.if_exception_type(
-                    exceptions.DeadlineExceeded,
                     exceptions.InternalServerError,
                     exceptions.ServiceUnavailable,
+                    exceptions.DeadlineExceeded,
```

Comment:
@crwilcox How is this still happening, given that the [googleapis `WORKSPACE` now uses `gapic-generator-0.35.9](https://github.com/googleapis/googleapis/commit/07d41a7e5cade45aba6f0d277c89722b48f2c956#diff-5493ff8e9397811510e780de47c57abb70137f1afe85d1519130dc3679d60ce5).

Reply:
Interestingly, PR #252 does *not* generate this mess.